stdlib/private/SwiftPrivateLibcExtras/SwiftPrivateLibcExtras.swift

```diff
@@ -25,9 +25,13 @@ public func _stdlib_mkstemps(_ template: inout String, _ suffixlen: CInt) -> CIn
   var utf8 = template.nulTerminatedUTF8
   let (fd, fileName) = utf8.withUnsafeMutableBufferPointer {
     (utf8) -> (CInt, String) in
-    let fd = mkstemps(UnsafeMutablePointer(utf8.baseAddress!), suffixlen)
-    let fileName = String(cString: utf8.baseAddress!)
-    return (fd, fileName)
+    return utf8.baseAddress!.withMemoryRebound(
+      to: CChar.self, capacity: Int(suffixlen)) {
```

Comment:
Sorry, we don't have a written document.

This would be the most complex case of wrapping the declaration:

```swift
func foo<
  T,
  U : Hashable  // Either put all type parameters on one line, on do one per line.
>( // Closing delimiters are put on a new line.
  arg1 x: T,
  arg2 y: U // Same, either everything fits on one line, or each gets its own line.
) -> Int
  where T.Z == U
{
...
}
```